short-response.md

```diff
@@ -121,10 +147,11 @@
   setLocation(address);
   console.log(`Our address is ${address}`);
   ```
-
+  - This logs `'829 Jefferson Ave.'` and `'Our address is 7 Marcus Garvey'` because when our function is invoked and given `address` only its value is passed to the function. Inside our function the parameter `location` is assigned this value, but reassiged to `'829 Jefferson Ave.'` and logged. Our `address` variable that is logged in the last line was not change because when we passed it to the function it was only by value and a copy was made for this purpose.
 
 13. **What do we mean when we say that JavaScript passes variables _by value_?**
-
+  
+    - This means the value of our variables is copied and pointed to by the variable that it was passed to. Our original variable still refereces the value it was assigned.
 
 14. **What does it mean to _pass by reference_? In what ways do arrays and objects appear to be passed by reference in JavaScript?**
-
+    - Passing by reference means the value passed to another variable or function is the address in memory to the value of our original variable. Arrays and objects demonstrate this when a varible is assigned another variable whose value is an object or array and making a change to the second varibale also mutates the first!
```

Comment:
# Question 14 Score:
## Technical
**2**: You nailed that PBR, passes an actual location in memory. However, PBR refers to the process of passing a variable to a _function_, not assigning a value to a variable.

## Writing
**3**: 👍